examples/structured_2d_dgsem/elixir_advection_basic_coupled.jl

```diff
@@ -0,0 +1,87 @@
+
+using OrdinaryDiffEq
+using Trixi
+
+
+###############################################################################
+# semidiscretization of the linear advection equation
+
+advectionvelocity = (1.0, 1.0)
```

Comment:
Please use a velocity not aligned with the diagonal, since this can have funny effects for DG schemes.

Reply:
We just rebuilt the `elixir_advection_basic.jl` with two meshes, each spanning half of the domain, to test if we get the exact same errors. Why is `elixir_advection_basic.jl` even using this advection velocity, then?

Reply:
Habit. But I like the argument that it should yield the same errors as the other implementation, so just leave it as it is.